pkgs/desktops/plasma-5/plasma-workspace/default.nix

```
@@ -60,6 +61,9 @@ mkDerivation {
       --replace 'ecm_query_qt(QtBinariesDir QT_INSTALL_BINS)' 'set(QtBinariesDir "${lib.getBin qttools}/bin")'
   '';
 
+  preFixup = "rm $out/bin/kcminit_startup";
+  postFixup = "ln -s $out/bin/kcminit $out/bin/kcminit_startup";
```

```suggestion
  preFixup = "
    rm $out/bin/kcminit_startup
  ";
  postFixup = "
    ln -s $out/bin/kcminit $out/bin/kcminit_startup
  ";
```
Those should always be multiline strings and please add a short comment why we do this.